client/packages/invoices/src/Prescriptions/LineEditView/hooks/usePrescriptionLineEditRows.ts

```diff
@@ -54,8 +57,9 @@ export const usePrescriptionLineEditRows = (rows: DraftStockOutLine[]) => {
   }, [allocatableRows, wrongPackSizeRows, onHoldRows, noStockRows]);
 
   const disabledRows = useMemo(() => {
+    if (isDisabled) return orderedRows;
```

Comment:
Fix for #6083

If the prescription is verified, it is `disabled`, so we make sure *all* rows are disabled.

Unfortunately adds another prop that must be passed down through about 3 levels 🙄